editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-element-renderer-utils.tsx

```diff
@@ -131,6 +135,10 @@ function monkeyUidProp(uid: string | undefined, propsToUpdate: MapLike<any>): Ma
   return monkeyedProps
 }
 
+function NoOpLookupRender(element: JSXElement, scope: MapLike<any>): React.ReactChild {
+  throw new Error('Utopia Error: createLookupRender was not used properly')
```

Comment:
It's worth including the element's name or type in the error here to help us debug if this does happen